lib/files.test.js

```diff
@@ -164,12 +165,51 @@ describe(_.startCase(filename), function () {
 
     it('returns result', function () {
       var filename = 'some-name',
+        result = 'some result',
+        expected = 'some result';
+
+      yaml.safeLoad.returns(temp2env(result));
```

Comment:
I don't think this "temp2env" library is under test, so it shouldn't be run in our tests.